File: VB/WindowsApplication1/MyDataSourceWrapper/MyObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports System.Collections

Namespace WindowsApplication1
	Public Class MyObject

		Public Sub New()

		End Sub

		Private _Value As Object
		Public Property Value() As Object
			Get
				Return _Value
			End Get
			Set(ByVal value As Object)
				_Value = value
			End Set
		End Property
	End Class
End Namespace

File: CS/WindowsApplication1/MyDataSourceWrapper/MyObject.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Collections;

namespace WindowsApplication1
{
    public class MyObject
    {

        public MyObject()
        {

        }

        private object _Value;
        public object Value
        {
            get { return _Value; }
            set { _Value = value; }
        }
    }
}
